src/Sql/Sql/Properties/Resources.resx

```diff
@@ -691,4 +691,46 @@
   <data name="ServerOutboundFirewallRuleFQDNDoesNotExist" xml:space="preserve">
     <value>Allowed FQDN with name '{0}' does not exist in the list of Outbound Firewall Rules (Allowed FQDNs) for Azure SQL Database server '{1}'.</value>
   </data>
+  <data name="ManagedInstanceLinkAlreadyExists" xml:space="preserve">
+    <value>Managed instance link with name '{0}' already exists on Azure Sql Managed Instance '{1}'</value>
+  </data>
+  <data name="ServerTrustCertificateAlreadyExists" xml:space="preserve">
+    <value>Certificate with name '{0}' already exists on Azure Sql Managed Instance '{1}'.</value>
+  </data>
+  <data name="RemoveAzureSqlInstanceServerTrustCertificateDescription" xml:space="preserve">
+    <value>Permanently removing Azure Sql Instance Server Trust Certificate in resource group '{0}' under managed instance '{1}' with name '{2}'.</value>
+  </data>
+  <data name="RemoveAzureSqlInstanceServerTrustCertificateWarning" xml:space="preserve">
+    <value>Are you sure you want to remove the Azure Sql Instance Server Trust Certificate in resource group '{0}' under managed instance '{1}' with name '{2}'?</value>
+  </data>
+  <data name="RemoveAzureSqlInstanceLinkDescription" xml:space="preserve">
+    <value>Permanently removing Azure Sql Managed Instance Link in resource group '{0}' under managed instance '{1}' with name '{2}'.</value>
+  </data>
+  <data name="RemoveAzureSqlInstanceLinkWarning" xml:space="preserve">
+    <value>Are you sure you want to remove the Azure Sql Instance Managed Instance Link in resource group '{0}' under managed instance '{1}' with name '{2}'?</value>
+  </data>
+  <data name="CreateAzureSqlInstanceServerTrustCertificateDescription" xml:space="preserve">
+    <value>Creating a new Server Trust Certificate in resource group '{0}' under managed instance '{1}' with name '{2}'.</value>
+  </data>
+  <data name="CreateAzureSqlInstanceServerTrustCertificateWarning" xml:space="preserve">
+    <value>Are you sure you want to create a new Server Trust Certificate in resource group '{0}' under managed instance '{1}' with name '{2}'?</value>
+  </data>
+  <data name="SetAzSqlDatabaseBackupShortTermRetentionInvalidParameters" xml:space="preserve">
+    <value>Invalid parameters, '{0}' and '{1}' can't both be null.</value>
+  </data>
```

Comment:
Added this string as a placeholder since it was missing from .resx file but it's being referenced in https://github.com/Azure/azure-powershell/blob/744f706c1d21cb075a98282a01f5db4ed246fdc5/src/Sql/Sql/Database%20Backup/Model/AzureSqlDatabaseBackupShortTermRetentionPolicyModel.cs#L56
Looks like it's been introduced in https://github.com/Azure/azure-powershell/pull/15325 so @lululilliancoding  can you confirm this string value is acceptable ? (or propose new)

Reply:
confirmed! Thanks for finding this.